### ISSUE TITLE
Empty tag "initContainers:" Cost Analyzer

### ISSUE THREAD

**Issue by michaelsatish** [kubecost/cost-analyzer-helm-chart#2259]
**Describe the bug**  
Empty tag "initContainers:" 

**To Reproduce**  
```sh
aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws

helm template kubecost \
  oci://public.ecr.aws/kubecost/cost-analyzer --version 1.103.3 \
  --namespace kubecost \
  -f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml | grep initContainers

# output from cost-analyzer deployment
initContainers:
```

The empty tag gets rendered because it is outside the conditional block.  
```yaml
# cost-analyzer-deployment-template.yaml
      initContainers:
{{- if .Values.supportNFS }}
```

**Expected behavior**  
Do not render empty tags.

**Comment by kwombach12** [comment]
Thanks @michaelsatish we will take a look at this

**Comment by jessegoodier** [comment]
This is fixed in 1.108.0, sorry for the delayed response. 
```
helm template kubecost \
  oci://public.ecr.aws/kubecost/cost-analyzer --version 1.108.0 \
  --namespace kubecost \
  -f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml | grep initContainers
```
